packages/truffle-core/lib/version.js

```diff
@@ -26,9 +26,10 @@ const logNode = (logger = console) => {
   logger.log(`Node ${process.version}`);
 };
 
-const logSolidity = (logger = console, versionInformation, config = {}) => {
+const logSolidity = (logger = console, versionInformation, config) => {
   let solcVersion;
   if (
+    config &&
```

Comment:
Cool! so will the else clause in line 39 just return the solc version that's shipped w/ truffle-core?

Reply:
Yes!